patches/minecraft/net/minecraft/client/model/HumanoidModel.java.patch

```diff
@@ -0,0 +1,52 @@
+--- a/net/minecraft/client/model/HumanoidModel.java
++++ b/net/minecraft/client/model/HumanoidModel.java
+@@ -264,6 +_,8 @@
+          case TOOT_HORN:
+             this.f_102811_.f_104203_ = Mth.m_14036_(this.f_102808_.f_104203_, -1.2F, 1.2F) - 1.4835298F;
+             this.f_102811_.f_104204_ = this.f_102808_.f_104204_ - ((float)Math.PI / 6F);
++         default:
++            this.f_102816_.applyTransform(this, p_102876_, net.minecraft.world.entity.HumanoidArm.RIGHT);
+       }
+ 
+    }
+@@ -304,6 +_,8 @@
+          case TOOT_HORN:
+             this.f_102812_.f_104203_ = Mth.m_14036_(this.f_102808_.f_104203_, -1.2F, 1.2F) - 1.4835298F;
+             this.f_102812_.f_104204_ = this.f_102808_.f_104204_ + ((float)Math.PI / 6F);
++         default:
++            this.f_102815_.applyTransform(this, p_102879_, net.minecraft.world.entity.HumanoidArm.LEFT);
+       }
+ 
+    }
+@@ -396,7 +_,7 @@
+    }
+ 
+    @OnlyIn(Dist.CLIENT)
+-   public static enum ArmPose {
++   public static enum ArmPose implements net.minecraftforge.common.IExtensibleEnum {
+       EMPTY(false),
+       ITEM(false),
+       BLOCK(false),
+@@ -416,5 +_,22 @@
+       public boolean m_102897_() {
+          return this.f_102890_;
+       }
++      // FORGE START
++      private net.minecraftforge.client.IForgeArmPose forgeArmPose;
```

Comment:
Should I move `//FORGE START` up? I'm unsure if modified original constructor should be included between `//FORGE START` and `//FORGE END`

Reply:
The marker should remain where it is, as it denotes the part of the class which is added by Forge. If one wanted to mark the modification to the original constructor, they'd use `// FORGE: `, but it's not needed here since the field it assigns is inside the block.